src/util/cloud_utils.js

```diff
@@ -170,7 +170,7 @@ function set_noobaa_s3_connection(sys) {
     const endpoint = system_address[0] && system_address[0].hostname;
     const access_key = sys.owner && sys.owner.access_keys && sys.owner.access_keys[0].access_key.unwrap();
     const secret_key = sys.owner && sys.owner.access_keys && sys.owner.access_keys[0].secret_key.unwrap();
-    dbg.log0('replication_server.set_noobaa_s3_connection: ', endpoint, access_key, secret_key);
+    dbg.log1('replication_server.set_noobaa_s3_connection: ', endpoint, access_key, secret_key);
```

Comment:
can you delete the secret_key from this print? I remove it in https://github.com/noobaa/noobaa-core/pull/7463 but it's was not code reviewed yet